src/app/pages/PerpetualPage/hooks/usePerpetual_OpenOrders.ts

```diff
@@ -34,6 +37,23 @@ type OpenOrdersHookResult = {
 export const usePerpetual_OpenOrders = (
   address: string,
 ): OpenOrdersHookResult => {
+  const completedTransactions = usePerpetual_completedTransactions();
```

Comment:
`Open Orders` table doesn't refresh automatically after new limit/stop trades so this probably doesn't work correctly.